### ISSUE TITLE
Manage Value Mappings in reusable lists

### ISSUE THREAD

**Issue by christianhauff** [grafana/grafana#83670]
**Why is this needed**:

I have several dashboards which include several tables containing IP-/MAC-Addresses in some places. I am using value mappings for each panel to translate these addresses into names for easier comprehension. As the value mappings are per panel, I have to manage all changes for all panels manually. I know I can do some sort of bulk editing when I edit my dashboard's JSON, but with every change, I have to manually change it for every panel on every dashboard.

**What would you like to be added**:

I could imagine some sort of central management or lists for value mappings that could be reused in panels. Defining a list of IP-Address-Mappings per dashboard or even globally would allow me to do changes one time and have them applied to all panels at once. I don't care if that would be an UI to manage that list or just a possibility to get the mappings from a data source.

**Who is this feature for?**

Every user who has reoccurring value mappings on the dashboards could probably use such a functionality.

**Comment by barneysspeedshop** [comment]
Yes, please work on this, grafana devs. Would love this functionality

**Comment by barneysspeedshop** [comment]
The issue is still relevant, it should not be closed without adding the functionality
